source/documentation/values/calculations.md

```diff
@@ -156,61 +156,119 @@ CSS to unitless numbers:
-[`min()`]: /documentation/modules/math#min
-[`max()`]: /documentation/modules/math#max
+[`round()`]: ../modules/math#round
+[`abs()`]: ../modules/math#abs
+[`min()`]: ../modules/math#min
+[`max()`]: ../modules/math#max
 
-If a `min()` or `max()` function call is a valid calculation expression, it will
+If a calculation function call is a valid calculation expression, it will
 be parsed as a calculation. But as soon as any part of the call contains a
 SassScript feature that isn't supported in a calculation, like the [modulo
-operator], it's parsed as a call to Sass's core `min()` or `max()` function
+operator], it's parsed as a call to the appropriate Sass math function
 instead.
 
 Since calculations are simplified to numbers when possible anyway, the only
 substantive difference is that the Sass functions only support units that can be
 combined at build time, so `min(12px % 10, 10%)` will throw an error.
 
-[modulo operator]: /documentation/operators/numeric
+[modulo operator]: /documentation/operators/numeric/
+
+#### `min()` and `max()`
+{% compatibility 'dart: ">=1.11.0 <1.42.0"', 'libsass: false', 'ruby: false', 'feature: "min and max syntax"' %}
+  LibSass, Ruby Sass, and versions of Dart Sass prior to 1.11.0 *always* parse
+  `min()` and `max()` as Sass functions. To create a plain CSS `min()` or
+  `max()` call for those implementations, you can write something like
+  `unquote("min(#{$padding}, env(safe-area-inset-left))")` instead.
+
+  CSS added support for [`min()` and `max()` functions] in Values and Units
+  Level 4, from where they were quickly adopted by Safari [to support the iPhoneX].
+  Since we already supported `min()` and `max()` as legacy Sass functions, we
+  had to implement logic for backwards-compatibility and for support as CSS
+  functions.
+
+  Versions of Dart Sass between 1.11.0 and 1.40.0, and between 1.40.1
+  and 1.42.0 parse `min()` and `max()` functions as [special functions] if
+  they're valid plain CSS, but parse them as Sass functions if they contain Sass
+  features other than interpolation, like variables or function calls.
+
+  Dart Sass 1.41.0 parses `min()` and `max()` functions as calculations, but
+  doesn't allow unitless numbers to be combined with numbers with units. This
+  was backwards-incompatible with the global `min()` and `max()` functions, so
+  that behavior was reverted.
+
+  [special functions]: /documentation/syntax/special-functions/
+{% endcompatibility %}
 
 {% headsUp %}
   Other calculations don't allow unitless numbers to be added to, subtracted
-  from, or compared to numbers with units. `min()` and `max()` are different,
+  from, or compared to numbers with units. [`min()`] and [`max()`] are different,
   though: for backwards-compatibility with the global Sass `min()` and `max()`
   functions which allow unit/unitless mixing for historical reasons, these units
   can be mixed as long as they're contained directly within a `min()` or `max()`
   calculation.
```

Comment:
This logic applies to `abs()` and single-argument `round()` as well, so we should probably move it up to the parent section.